bots/image-refresh

```diff
@@ -279,7 +279,7 @@ class ImageTask(object):
         # request anyway, for extra attention
 
         if ret != 0:
-            os.unlink("images/" + self.image)
+            os.unlink(os.path.join(BOTS, "images", self.image))
```

Comment:
Are we missing another PR for this? Current master still has the images in `./test/images/`, not in `./bots/images/`?